DECISIONLOG.md

```diff
@@ -30,4 +30,27 @@ tests:
   ad_reporting:
     not_null_ad_reporting__account_report_account_id:
       +enabled: false
-```
\ No newline at end of file
+```
+
+### Timezone Considerations
+
+The table below documents timezone differences across platforms in the `dbt_ad_reporting` package. These differences exist due to the following reasons:
+
+- Ad platforms send pre-aggregated data that cannot be back-calculated to a finer granularity to account for timezone differences. 
+- Some platforms offer hourly report data, which could potentially be standardized. However, this approach does not ensure full coverage due to non-standard timezones, such as ±30-minute and ±45-minute offsets.
+
+Although this presents challenges within this package, customers can achieve standardization by configuring all Ad accounts and connectors to UTC whenever possible.
+
+| Platform | Grain of Data | Timezone | Hourly Custom Reports? | Can Configure Time Zones for Custom Reports? |
+|----------|---------------|----------|------------------------|----------------------------------------------|
+| Amazon Ads | Day | CLIENT SET | No | No |
+| Apple Search Ads | Day | CLIENT SET | Yes | ORTZ, UTC |
+| Facebook Ads | Day | CLIENT SET | No | No |
+| Google Ads | Hour - aggregated to daily in the package | CLIENT SET | Yes | No |
+| Linkedin Ad Analytics | Day | UTC | No | No |
+| Microsoft Advertising | Hour - aggregated to daily in the package | CLIENT SET | No | No |
+| Pinterest Ads | Hour - aggregated to daily in the package | UTC | Yes | No |
+| Reddit Ads | Day | UTC | No | Customer can choose timezone in custom reports |
+| Snapchat Ads | Hour | UTC | No | No |
+| TikTok Ads | Hour | UTC | Yes | No |
+| Twitter Ads | Day | CLIENT SET | No | No |
```

Comment:
I initially included the longer explanation from our table detailing whether the timezone is UTC or client-set. However, I decided to simplify it to just 'CLIENT SET.' While the detailed information was specific, I was concerned about how quickly it might become outdated. I believe this still conveys the necessary context, but let me know if I should add it back.

Reply:
That's a good point, but I'm wondering how often this will actually change. I think a good compromise could be splitting it into two categories - `CLIENT SET (in Fivetran)` vs `CLIENT SET (in Ad Platform)`. This way, they know at least _where_ to go if they want to change the timezone. 

We could also preface this section with "As of January 2025..."